samples/data-sealing/common/shared.h

```diff
@@ -4,6 +4,8 @@
 #ifndef _SHARED_H
 #define _SHARED_H
 
+#include <stddef.h>
```

Comment:
This was required since we use a `size_t` below, not sure how it built without it previously.

Reply:
Ah, @anakrish I knew there was a header ordering issue. Still, IWYU.